CONTRIBUTING.md

```diff
@@ -135,56 +135,31 @@ In README files this ends up being slightly different. Because the version tag i
     ```
     [//]: # ({x-version-update-end})
 
-What if I've got something that, for whatever reason, shoudln't be updated? There's a tag for that.
-
-`<!-- {x-version-exempt;<groupId>:<artifactId>;reason for excemption} -->`
-
-In theory, absence of an x-version-update tag would do the same thing but the tooling is still being developed and eventually there will be checkin blockers if xml has a version element with no tag.
-
 ### What does the process look like?
 
 Let's say we've GA'd and I need to tick up the version of azure-storage libraries how would I do it? Guidelines for incrementing versions after release can be found [here](https://github.com/Azure/azure-sdk/blob/master/docs/policies/releases.md#incrementing-after-release).
 
-1. I'd open up eng\versioning\version_client.txt and update the current-versions of the libraries that are built and released as part of the azure storage pipeline. This list can be found in pom.service.xml under the sdk/storage directory. It's worth noting that any module entry starting with "../" are external module dependencies and not something that's released as part of the pipeline. Once we GA, these build dependencies for library components outside of a given area should go away and be replaced with downloading the appropriate dependency from Maven like we do for external dependencies.
+1. I'd open up eng\versioning\version_client.txt and update the current-versions of the libraries that are built and released as part of the azure storage pipeline. This list can be found in pom.service.xml under the sdk/storage directory. It's worth noting that any module entry starting with "../" are external module dependencies and not something that's released as part of the pipeline. Dependencies for library components outside of a given area be downloading the appropriate dependency from Maven like we do for external dependencies.
```

Comment:
`outside of a given area be downloading`    hange to `outside of a given area would be downloading`

Reply:
Done. Updated with the latest PR.